config/200-channelable-clusterrole.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Use this aggregated ClusterRole when you need read/write access to "Channelables"
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: channelable
```

Comment:
How about `channelable-manipulator`? It feels like there should be describing what is done with the channelables.

Reply:
Hmm, by looking at other ClusterRoles, we are not saying what we do with them. For example, the `eventing-broker-filter`, etc. The only one is the `addressable-resolver`, which I guess is the only aggregated one besides this. 
I'm more into decoupling the name from the verbs. But if you prefer it as `channel-manipulator` I can do the change. In that case, you want me to also change `imc-channelable` to `imc-channelable-manipulator` and so on?

Reply:
I don't feel strongly about it, but if we want to describe it more, I think I'd prefer something like: channelable-admin?

Reply:
`channelable-admin` sounds a little off to me because it doesn't have all the permissions. How about `channelable-director`?

And yes, this name change should occur on all the roles that get aggregated into this one too.

Reply:
Changed it to `channelable-manipulator`. Thanks!